.azure-pipelines/build-swss-template.yml

```diff
@@ -138,6 +145,33 @@ jobs:
       patterns: |
         libdashapi*.deb
     displayName: "Download dash api test"
+  - task: DownloadPipelineArtifact@2
+    inputs:
+      source: specific
+      project: build
+      pipeline: sonic-net.sonic-platform-vpp
+      artifact: vpp
+      runVersion: 'latestFromBranch'
+      runBranch: 'refs/heads/main'
+      allowPartiallySucceededBuilds: true
+      path: $(Build.ArtifactStagingDirectory)/download
+    displayName: "Download sonic platform-vpp deb packages"
+    condition: eq('${{ parameters.arch }}', 'amd64')
+  - script: |
+      set -ex
+      sudo dpkg -i download/libvppinfra-dev_*_${{ parameters.arch }}.deb
+      sudo dpkg -i download/libvppinfra_*_${{ parameters.arch }}.deb
+      sudo dpkg -i download/vpp_*_${{ parameters.arch }}.deb
+      sudo dpkg -i download/vpp-crypto-engines_*_${{ parameters.arch }}.deb
+      sudo dpkg -i download/vpp-dbg_*_${{ parameters.arch }}.deb
+      sudo dpkg -i download/vpp-dev_*_${{ parameters.arch }}.deb
+      sudo dpkg -i download/vpp-plugin-core_*_${{ parameters.arch }}.deb
+      sudo dpkg -i download/vpp-plugin-devtools_*_${{ parameters.arch }}.deb
+      sudo dpkg -i download/vpp-plugin-dpdk_*_${{ parameters.arch }}.deb
+      sudo dpkg -i download/python3-vpp-api_*_${{ parameters.arch }}.deb
```

Comment:
suggestion is to add these dependency here. https://github.com/sonic-net/sonic-buildimage/tree/master/platform/vs/docker-sonic-vs